src/renderer/ipc.ts

```diff
@@ -38,10 +36,18 @@ function getImplementation(): IPCInterface {
   };
 }
 
+let cachedInterface: IPCInterface | undefined;
+
 /**
  * Renderer IPC interface
  *
  * Favours IPC if its been exposed via a preload script but will
  * fallback to custom protocol and fetch is IPC is not available
  */
-export const IPC: IPCInterface = getImplementation();
+export function getIPC(): IPCInterface {
+  if (!cachedInterface) {
+    cachedInterface = getImplementation();
```

Comment:
Will there be any worries about the cache getting stale? Like some condition where `window.__SENTRY_IPC__` somehow changes?

Reply:
We were only ever calling `getImplementation()` once before. This change just does it lazily rather than at parse time.